src/styles/_catalog.scss

```
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms;
```

don't use all, it's bad for performance

```suggestion
    transition: transform 300ms, color 300ms;
```